saleor/graphql/account/types.py

```diff
@@ -92,12 +93,18 @@ class AddressInput(BaseInputObjectType):
             "[libphonenumber](https://github.com/google/libphonenumber) library."
         )
     )
-
     metadata = graphene.List(
         graphene.NonNull(MetadataInput),
         description="Address public metadata." + ADDED_IN_315,
         required=False,
     )
+    skip_validation = graphene.Boolean(
+        description="Determine if the address should be validated."
```

Comment:
I would add here a note about required permission. Also maybe a single sentence about what exactly means the validation ( I mean, with skip-validation, we will not validate the fields like postal-code and require a specific format of the input etc)